### DISCUSSION TITLE
How to upgrade SQLite version

### DISCUSSION THREAD

**Discussion by pme442** [ericsink/SQLitePCL.raw#599] [Q&A] [answered]
I have a cross-platform Xamarin Forms app.
I also have a .NET Maui app that is build in .NET 8 which is showing me the same results.

When I run the following code, it returns: sqlversion = 3.39.2:
string query = string.Format(@"SELECT SQLITE_VERSION()");
var sqlversion = _db.ExecuteScalar(query);
Debug.WriteLine("sqlversion = " + sqlversion);

All of my Nuget packages are updated to the latest. For example, in my iOS.csproj:
sqlite-net-pcl = 1.9.172
sqlite-net-sqlcipher = 1.9.172
SQLitePCLRaw.bundle_e_sqlcipher = 2.1.8
SQLitePCLRaw.bundle_green = 2.1.8

Where is it getting 3.39.2 from?
Is there something else I need to update in order to bump it up?
I need it to be at 3.44.0 or higher.

**Comment by ericsink** [comment, marked as answer]
BTW, you should only have one bundle package.

Anyway, you're getting the version number from e_sqlcipher, which is not as current:

https://github.com/ericsink/SQLitePCL.raw/wiki/Expectation-setting-for-e_sqlcipher

**Reply by pme442** [reply]
Yeah, I just noticed that I'm only using the SQLitePCLRaw.bundle_e_sqlcipher in the iOS project.  I have the other one in the UWP project.
OK thanks, Eric.
